astropy/modeling/tests/test_models.py

```diff
@@ -690,11 +689,46 @@ def test_tabular_with_bounding_box():
     t = models.Tabular1D(points, values)
     result = t(1, with_bounding_box=True)
 
+    assert result == 3.4
+    assert t.inverse(result, with_bounding_box=True) == 1.
+
 
 @pytest.mark.skipif("not HAS_SCIPY_14")
-def test_bounding_box_with_units():
+def test_tabular_bounding_box_with_units():
     points = np.arange(5)*u.pix
     lt = np.arange(5)*u.AA
     t = models.Tabular1D(points, lt)
+    result = t(1*u.pix, with_bounding_box=True)
+
+    assert result == 1.*u.AA
+    assert t.inverse(result, with_bounding_box=True) == 1*u.pix
+
+
+@pytest.mark.skipif("not HAS_SCIPY_14")
+def test_tabular1d_inverse():
+    """Test that the Tabular1D inverse is defined"""
+    points = np.arange(5)
+    values = np.array([1.5, 3.4, 6.7, 7, 32])
+    t = models.Tabular1D(points, values)
+    result = t.inverse((3.4, 6.7))
+    assert_allclose(result, np.array((1., 2.)))
 
-    assert(t(1*u.pix, with_bounding_box=True) == 1.*u.AA)
+    # Check that it works for descending values in lookup_table
+    t2 = models.Tabular1D(points, values[::-1])
+    result2 = t.inverse((3.4, 6.7))
```

Comment:
I think if you add a check here that `t.inverse.points = t.lookup_table[::1]` the code coverage test will pass